tests/test_flow/test_persistence.py

```diff
@@ -821,3 +821,58 @@ def y():
 
         flow = builder.build()
         assert flow.get('y') == 1
+
+
+def test_volatile_ancestor(builder):
```

Comment:
Looks like we don't have a test for the interaction between `@volatile` and `@memoize`?

Reply:
Based on another comment thread, I guess I can add it and test if it throws error.